servicemesh-install/topics/install.adoc

```diff
@@ -1,21 +1,227 @@
+
+|
+|`password`
+|OpenShift user password for running Fabric8 launcher.
+|`developer`
+
+|github
+|`username`
+|Should be modified to reflect the  https://help.github.com/articles/signing-up-for-a-new-github-account/[GitHub account] you want to use to run the Fabric8 launcher.
+|N/A
+
+|
+|`token`
+|GitHub https://github.com/settings/tokens[personal access token] you want to use to run the Fabric8 launcher.
+|N/A
+
+|catalog
+|`filter`
+|Filter to apply to the Red Hat booster catalog.
+|booster.mission.metadata.istio
+
+|
+|`branch`
+|Version of the Red Hat booster catalog that should be used with Fabric8.
+|v62
+
+|
+|`repo`
+|GitHub repo to use for Red Hat booster catalog.
+|https://github.com/fabric8-launcher/launcher-booster-catalog.git
+|===
+
+
+[[installing-operator]]
+== Installing the operator
 ////
 TODO
 Add an overview of Operators
 ////
-The {ProductShortName} installation process introduces a Kubernetes operator to manage the installation of the control plane within the `istio-system` namespace.  This operator defines and monitors a custom resource related to the deployment, update, and deletion of the control plane.
+The {ProductShortName} installation process introduces a Kubernetes _operator_ to manage the installation of the control plane within the `istio-system` namespace.  This operator defines and monitors a custom resource related to the deployment, update, and deletion of the control plane.
 
-[[installing-operator]]
-== Installing the operator
-The following steps will install the {ProductShortName} operator into an existing {product-title} 3.10 installation; they can be executed from any host with access to the cluster.  Please ensure you are logged in as a cluster admin before executing the following commands.
+You can find the https://github.com/Maistra/openshift-ansible/tree/maistra-0.2.0-ocp-3.1.0-istio-1.0.2/istio[operator templates on GitHub].
+
+[NOTE]
+====
+The custom resource *must* be called `istio-installation`, that is, the metadata value for `name` must be `istio-installation` and it *must* be installed into the `istio-operator` namespace that is created by the operator.
+====
 
-You can find the https://github.com/Maistra/openshift-ansible/tree/maistra-0.1.0-ocp-3.1.0-istio-1.0.0/istio[operator templates on GitHub].
+The following steps will install the {ProductShortName} operator into an existing {product-title} 3.10 installation; they can be executed from any host with access to the cluster.  Please ensure you are logged in as a cluster admin before executing the following commands.
 
 ```
 $ oc new-project istio-operator
 $ oc new-app -f istio_product_operator_template.yaml --param=OPENSHIFT_ISTIO_MASTER_PUBLIC_URL=<master public url>
 ```
+[NOTE]
+===
+The OpenShift Master Public URL should be configured to match the public URL of your OpenShift Console, this parameter is required by the Fabric8 Launcher.
+===
+
+
 [[verifying-operator-installation]]
 == Verifying operator installation
 
```

Comment:
Can't comment on it since it's not part of this PR, but see line 256. "To deploy the control plan,". Plan should be plane.

Reply:
Also, lines 245-250. The only useful part of this is line 250. We're trimming that down in community docs. May be useful to do here as well.